JythonCli.java

```diff
@@ -91,8 +91,9 @@ void initEnvironment(String[] args) throws IOException {
             System.exit(1);
         }
 
+        scriptFilename = "";
         for (String arg : args) {
-            if (arg.endsWith(".py")) {
+            if (scriptFilename.isEmpty() && arg.endsWith(".py")) {
```

Comment:
I would use `scriptFilename==null` here as we signify absence of a script file that way later on in ` main`, and `=""` will disable that.

Reply:
It's hard writing correct code!